src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/GetAzureVpnClientIpsecParametersCommand.cs

```diff
@@ -1,49 +0,0 @@
-﻿// ----------------------------------------------------------------------------------
-//
-// Copyright Microsoft Corporation
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-// http://www.apache.org/licenses/LICENSE-2.0
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-// ----------------------------------------------------------------------------------
-
-using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
-using System.Management.Automation;
-using MNM = Microsoft.Azure.Commands.Network.Models;
-
-namespace Microsoft.Azure.Commands.Network.VirtualNetworkGateway
-{
-    [Cmdlet(VerbsCommon.Get, "AzureRmVpnClientIpsecParameter"), OutputType(typeof(MNM.PSVpnClientIPsecParameters))]
```

Comment:
Why is the cmdlet being removed? I still see it in the psd1 file, and we cannot allow breaking changes until the next breaking change release.

Reply:
have no idea why Evgeny pushed it to this branch, I'll close it and created new one. Please review 
https://github.com/Azure/azure-powershell/pull/6347

Reply:
This is [removal PR](https://github.com/Azure/azure-rest-api-specs/pull/2975/files) to API specs so removing this methods should be expected.

Reply:
@EvgenyAgafonchikov  We have no approval from PS and no actions from contributors so no need to save this removal if it works...